mappings_test.go

```diff
@@ -0,0 +1,32 @@
+package sourcemap
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestParseMappings(t *testing.T) {
+	t.Parallel()
+	cases := map[string][]mapping{
+		";;;;;;kBAEe,YAAY,CAC1B,C;;AAHD": {
+			{genLine: 7, genColumn: 18, sourceLine: 3, sourceColumn: 15, namesInd: -1},
+			{genLine: 7, genColumn: 30, sourceLine: 3, sourceColumn: 27, namesInd: -1},
+			{genLine: 7, genColumn: 31, sourceLine: 4, sourceColumn: 1, namesInd: -1},
+			{genLine: 7, genColumn: 32, sourceLine: 4, sourceColumn: 1, namesInd: -1},
+			{genLine: 9, genColumn: 0, sourceLine: 1, sourceColumn: 0, namesInd: -1},
```

Comment:
This line says that 9:0 should be mapped to 1:0 which does not make much sense...

Reply:
it's what babel generated and https://sokra.github.io/source-map-visualization/ works with it :shrug: it is actually pointing to the first line and the line is `import ....` so :shrug:

Reply:
Okay, I guess generators are free to reorder the code...